config/crd/bases/crd.cloud.antrea.io_cloudentityselectors.yaml

```diff
@@ -41,6 +41,9 @@ spec:
               accountName:
                 description: AccountName specifies cloud account in this CloudProvider.
                 type: string
+              accountNamespace:
```

Comment:
Please under the crds in helm dir too.. build/charts/nephe/charts/crd

Reply:
Copy this generated file to build/charts/nephe/charts/crds/cloudentityselectors.yaml?

Reply:
No, manually patch it.

Reply:
done